micrometer/prometheus/src/test/java/io/quarkus/ts/openshift/micrometer/PrimeNumberResourceOpenShiftIT.java

```diff
@@ -22,7 +21,7 @@
  * - `prime_number_max_{uniqueId}`: max prime number that is found
  * - `prime_number_test_{uniqueId}`: with information about the calculation of the prime number
  */
-@OpenShiftTest(strategy = UsingQuarkusPluginDeploymentStrategy.class)
+@OpenShiftTest
```

Comment:
I'm curious - what implies this change?

Reply:
None. It should behave the same, but the official one is the defaults strategy.